pkg/snapshot/filesystem.go

```diff
@@ -548,7 +558,40 @@ func shouldResetFileSystemMount(ctx context.Context, clientset kubernetes.Interf
 	}
 
 	if err := k8sutil.WaitForPod(ctx, clientset, deployOptions.Namespace, checkPod.Name, time.Minute*2); err != nil {
-		finalErr = errors.Wrap(err, "failed to wait for file system minio check pod to complete")
+		genericErr := errors.Wrap(err, "failed to wait for file system minio check pod to complete")
```

Comment:
what about the reset pod and the keys SHA pod? i see this PR will only return errors for the check pod

Reply:
I think those will still show a generic error and logs will be in the support bundle....  This is the only one I tested.